core-web/apps/dotcms-ui/src/app/view/components/dot-link/dot-link.component.scss

```diff
@@ -0,0 +1,25 @@
+@use "variables" as *;
+
+a,
+a:focus,
+a:active {
+    text-decoration: none;
+}
+
+// This styles are the same as .p-button.p-button-text in _buttons.scss,
+// the problem is the :enabled pseudoclass, hyperlink tag doesn't have that pseudoclass
+
+.p-button {
+    &:hover {
+        background: $color-palette-primary-100;
+    }
+
+    &:focus {
+        background: white;
+        box-shadow: $md-shadow-1;
```

Comment:
We're removing this.

Reply:
done https://github.com/dotCMS/core/pull/24807/commits/e2fe9bd3656d5d95428847c2b7389c64979457c3#diff-b9de41344c7150e006f9bb8d4465c2154bcb2c92f875a9f22013481667a335f7L19